jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
Can you explain what's critical about it? In windows we set one of the [DNS server to be localhost](https://github.com/pivotal-cf-experimental/garden-windows-bosh-release/blob/master/jobs/garden-windows/templates/pre-start.ps1#L60) for consul and everything works. Arguably we could move that part of the setup to a `pre-start` script for consul instead of garden-windows